sonar-go-checks/src/test/resources/checks/TodoCommentCheck/todo_comment.go

```diff
@@ -48,4 +49,12 @@ TODO Multiline comment */
 // valid end of line todo
 
 // Noncompliant@+1
-// valid end of file todo
\ No newline at end of file
+// valid end of file todo
```

Comment:
This one should be moved at the end of file.
I believe the goal of this test is to ensure that we don't skip comment that are at the end of file, which could be the case as some analyzer attach comment to next token, which could be an issue at the end of file (no next token).